### ISSUE TITLE
Simplify TLS certificate directions

### ISSUE THREAD

**Issue by PeterEltgroth** [pivotal-cf/docs-build-service#40]
https://github.com/pivotal-cf/docs-build-service/blob/1a0114c2e81adc2226aa2c0c42651c9d9dc9e23d/installing.html.md.erb#L94

We manually generated out own cert following the directions in [Director SSL Certificate Configuration with OpenSSL](https://bosh.io/docs/director-certs-openssl/#generate) and then added the cert to Bosh Director like shown in the [Harbor docs](https://docs.pivotal.io/partners/vmware-harbor/integrating-pks.html#load-harbor-cert). This seems more approachable than modifying the Director deployment manifest.

**Comment by jncd** [comment]
Hi @PeterEltgroth (Peter),
Thanks for this info. I've reached out to Ashwin Venkatesh (in the #tanzu-build-service channel) and I'll pair with him next week on improving the directions.

**Comment by PeterEltgroth** [comment]
Sorry for the slow reply, I was out of office shortly after submitting this

It is interesting that TBS worked for us if that isn't quite the same thing and it's been a while, so I don't recall all the steps we took anymore. We did also install a Cert for Harbor and plugged our TBS deployment into a Concourse pipeline and successfully tested it.

In any case, eliminating the need for the TLS certificate process will be a usability win. 

Thanks
